vertical-pod-autoscaler/docs/installation.md

```diff
@@ -10,12 +10,13 @@
   - [Install command](#install-command)
   - [Tear down](#tear-down)
 
-The current default version is Vertical Pod Autoscaler 1.2.1
+The current default version is Vertical Pod Autoscaler 1.2.2
 
 ## Compatibility
 
 | VPA version     | Kubernetes version |
 |-----------------|--------------------|
+| 1.2.2           | 1.27+              |
```

Comment:
I'm wondering: should we even do this table for patch versions? If the compatibility would change, this probably couldn't be released as a patch. So maybe we just make this `1.2.x` and `1.1.x` instead?

Reply:
Yeah good call, done.

Also done in #7700.